domains/osenoye.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "erickvenom99",
+    "email": "osenoye.aghaulor@gmail.com"
+  },
+  "records": {
+    "TXT": "google-site-verification=XQ2SWp81z61JnPZlPu-kAe4BEHwT5PNep5OKK0A-uGQ"
```

you forget the CNAME
```suggestion
    "CNAME": "ghs.googlehosted.com",
    "TXT": "google-site-verification=XQ2SWp81z61JnPZlPu-kAe4BEHwT5PNep5OKK0A-uGQ"
```